pkg/docker/docker_test.go

```diff
@@ -0,0 +1,212 @@
+package docker_test
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+
+	"github.com/testground/testground/pkg/docker"
+	"github.com/testground/testground/pkg/rpc/rpctest"
+)
+
+var (
+	cli *client.Client
+)
+
+func init() {
```

Comment:
This is a candidate for `TestMain` if we ever need some kind of teardown - https://golang.org/pkg/testing/#hdr-Main

Reply:
Yup. I don't think I ever investigated when `init()` vs. `t.Main()` run.